packages/platform-sdk-ark/package.json

```diff
@@ -33,7 +33,7 @@
 		"@arkecosystem/platform-sdk-crypto": "workspace:*",
 		"@arkecosystem/platform-sdk-intl": "workspace:*",
 		"@arkecosystem/platform-sdk-support": "workspace:*",
-		"dot-prop": "^5.0.0",
+		"dot-prop": "^6.0.1",
```

Comment:
I don't remember the specific issue anymore but some packages broke with 6.0 because of some changes to how they read some objects.

Reply:
Ok. Rollingback.